src/main/resources/application.properties

```
@@ -1,7 +1,7 @@
 spring.datasource.url=jdbc:postgresql://localhost:5432/KindHandsBot
 spring.datasource.username=
 spring.datasource.password=
-spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.hibernate.ddl-auto=update
```

```suggestion
spring.jpa.hibernate.ddl-auto=validate
```
validate надо использовать, а не update, иначе смысла нет никакого в liquibase - можно его удалить из проекта тогда